src/scripts/timezone.test.js

```diff
@@ -196,5 +234,27 @@ describe("Handy Tau-bot timezone conversions", () => {
 
       expect(slack.postEphemeralMessage).not.toHaveBeenCalled();
     });
+
+    it("does not respond to an invalid time", async () => {
+      message.event.text = "25:12";
+      await handler(message);
+
+      expect(slack.postEphemeralMessage).not.toHaveBeenCalled();
+    });
+
+    // currently failing but may be worth investigating in future
+    it.skip("correctly converts 24 hour time", async () => {
+      message.event.text = "23:42 cst";
+      await handler(message);
+
+      expect(slack.postEphemeralMessage).toHaveBeenCalledWith(
+        responseFor("user ny", "12:42 est")
```

Comment:
```suggestion
        responseFor("user ny", "12:42")
```

The response wouldn't include a timezone. I think that might be why this test is failing.

Reply:
Yep, that was it!  Thanks for the catch :)